client/lib/site/utils.js

```diff
@@ -176,3 +177,17 @@ export function hasSiteFeature( site, feature ) {
 		return planHasFeature( site.plan.product_slug, feature );
 	}
 }
+
+/**
+ * Checks if the site is eligible for SEO features.
+ *
+ * @param {object} site Site to check
+ * @param {string} state State tree
+ * @param {string} siteId Site id
+ * @returns {boolean} True if does
+ */
+export function isEligibleForSEOFeatures( site, state, siteId ) {
+	if ( site && site.plan && state ) {
```

Comment:
Nitpick: Show we return false when this check doesn't pass?

Reply:
Good point, changed here: https://github.com/Automattic/wp-calypso/pull/53196/commits/188710bebb54c56996e0266435a6b2dab5ae487f